README.md

```diff
@@ -3,8 +3,9 @@
 [![codecov](https://codecov.io/gh/ucsb-cs156-w22/team02-w22-7pm-2/branch/main/graph/badge.svg?token=rZT26BG1Dj)](https://codecov.io/gh/ucsb-cs156-w22/team02-w22-7pm-2)
 
 Storybook is here:
-* Production: <https://ucsb-cs156-w22.github.io/starter-team02-docs/>
-* QA:  <https://ucsb-cs156-w22.github.io/starter-team02-docs-qa/>
+
+- Production: <https://ucsb-cs156-w22.github.io/team02-w22-7pm-2-docs/>
+- QA: <https://ucsb-cs156-w22.github.io/team02-w22-7pm-2docs-qa/>
```

Comment:
A small mistake here (just missing a dash after 7pm-2)
should be https://github.com/ucsb-cs156-w22/team02-w22-7pm-2-docs-qa